src/models/task/index.js

```diff
@@ -39,6 +39,10 @@ const TaskModel = (repo) => {
         const [task, err] = await repo.deleteTask(taskID);
         return [task, err];
     };
+    const editTask = async (taskID, updatedTask) => {
+        const [task, err] = await repo.editTask(taskID, updatedTask);
```

Comment:
Nit but you can do `return await repo.editTask(taskID, updatedTask);` No need for the extra variable declaration